framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
Does "log out" mean "terminate this session", or "terminate this *authenticated* session"? If it's the latter, removing the session from the DB might not make sense.